sg/logger.go

```diff
@@ -18,11 +17,11 @@ func NewLogger(name string) *log.Logger {
 	prefix := name
 	prefix = strings.TrimPrefix(prefix, "main.")
 	prefix = strings.TrimPrefix(prefix, "go.einride.tech/sage/tools/")
-	// Strip namespace from name.
+
+	// Separate namespace and target with colon, expecting the string to be
+	// of the format `namespace.target`.
 	if len(strings.Split(prefix, ".")) > 1 {
-		if unicode.IsUpper([]rune(strings.Split(prefix, ".")[0])[0]) {
-			prefix = strings.Join(strings.Split(prefix, ".")[1:], "")
-		}
```

Comment:
I'm not sure if we need to do this `IsUpper` check now, the join just seems to work for the targets I test with. Do you know @odsod if I'm missing something?

Reply:
Your update makes sense - let's see if there's an edge case that pops up, but looks good to me!